.github/workflows/playwright.yml

```
@@ -0,0 +1,41 @@
+name: Playwright Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Let's run this on all pushes and pull requests.

```suggestion
on:
  workflow_dispatch:
  push:
  pull_request:
```